### ISSUE TITLE
Fancybox-media helper is not working

### ISSUE THREAD

**Issue by morriz17** [fancyapps/fancyBox#965]
I use to used fancybox from long time ago and I just made an excercise and media helper is not working, can you tell me if I did something wrong please.

This is in my head:

<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>

<link rel="stylesheet" href="fancybox/source/jquery.fancybox.css?v=2.1.5" type="text/css" media="screen" />

<script type="text/javascript" src="fancybox/source/jquery.fancybox.pack.js?v=2.1.5"></script>

<link rel="stylesheet" href="fancybox/source/helpers/jquery.fancybox-buttons.css?v=1.0.5" type="text/css" media="screen" />

<script type="text/javascript" src="fancybox/source/helpers/jquery.fancybox-buttons.js?v=1.0.5"></script>

<script type="text/javascript" src="fancybox/source/helpers/jquery.fancybox-media.js?v=1.0.6"></script>

<link rel="stylesheet" href="fancybox/source/helpers/jquery.fancybox-thumbs.css?v=1.0.7" type="text/css" media="screen" />

<script type="text/javascript" src="fancybox/source/helpers/jquery.fancybox-thumbs.js?v=1.0.7"></script>

<script type="text/javascript" src="fancybox/lib/jquery.mousewheel-3.0.6.pack.js"></script>

This is my Javascript:

<script type="text/javascript">
        $(document).ready(function() {
                    $(".fancybox-thumb").fancybox({
                    openEffect  : 'elastic',
                    closeEffect : 'fade',
                    prevEffect  : 'swing',
                    nextEffect  : 'swing',
                    'loop': false,
                    padding: 0,
                    helpers : {
                    title   : {
                        type: 'outside',
                        position: 'bottom'
                    },
                    thumbs  : {
                        width   : 75,
                        height  : 50,
                        position : 'top'
                    },
                     overlay : {
                        css : {
                            'background' : 'rgba(255, 42, 45, 0.95)'
                        }
                    }
                }
            });
            $('.fancybox-media').fancybox({
                openEffect  : 'swing',
                closeEffect : 'swing',
                helpers : {
                    media : {}
                }
            });
        });
</script>

This is my body:
<div class="fotos"><a class="fancybox-media" href="https://www.youtube.com/watch?v=kso8Tk9QE_E"><img src="http://farm3.staticflickr.com/2870/8985207189_01ea27882d_m.jpg" width="267" height="150" /></a>
        </div>
        <div class="fotos"><a class="fancybox-media" href="http://maps.google.com/maps?q=Eiffel+Tower,+Avenue+Gustave+Eiffel,+Paris,+France&t=h&z=17" title="London Eye"><img src="http://farm4.staticflickr.com/3677/8962691008_7f489395c9_m.jpg" width="267" height="150" /></a>
        </div>

**Comment by fancyapps** [comment]
Hi, please, upgrade to v3.

**Comment by MichiyoYo** [comment]
the most useless answer

**Comment by fancyapps** [comment]
What did you expect? 1) The author of this issue did not provide any useful information about this issue, some random peaces of code are not sufficient, because 2) it just works fine as you can see on homepage and on any other demo.
